### ISSUE TITLE
[BUG] Button text is bold in iOS for buttons with type="submit"

### ISSUE THREAD

**Issue by KemalPajevic** [kirbydesign/designsystem#3417]
- [X] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
If you have a kirby-button that has `type="submit"` then the button text will be bold in iOS. This is the case both in an iOS app and in Safari. On desktop this is not an issue, neither in Chrome nor Safari. I have not tested in Android.

## Describe how to reproduce the bug

1. Add a kirby-button to a component
2. Set it's type to `type="submit"`
3. Open the component in iOS
4. See that the button text is bold

## Which Kirby version was used?
9.2.1

## What was the expected behavior?
The button text has regular weight.

## Add any screenshots
![IMG_5424](https://github.com/kirbydesign/designsystem/assets/9060580/775b4bab-6ed9-43e4-b533-3e25d3955507)

## Please complete the following information:
- OS: iOS 17.3.1
- Browser: Safari

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
